client/rest-high-level/src/test/java/org/elasticsearch/client/transform/transforms/latest/LatestDocConfigTests.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.client.transform.transforms.latest;
+
+import org.elasticsearch.common.xcontent.XContentParser;
+import org.elasticsearch.search.sort.SortBuilders;
+import org.elasticsearch.test.AbstractXContentTestCase;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.function.Predicate;
+
+public class LatestDocConfigTests extends AbstractXContentTestCase<LatestDocConfig> {
```

Comment:
there are also test classes for hlrc <-> server and back in the hlrc sub package. Every new class should implement this, although we haven't reached full coverage yet. Adding support for latest should be easy.

Reply:
Done.